### DISCUSSION TITLE
Usage example

### DISCUSSION THREAD

**Discussion by susanin-ivan** [ilian6806/stable-diffusion-webui-eyemask#6] [Q&A] [answered]
Hello ilian6806,

Much appreciate your work and investment in developement of new usefull features!

I've installed this extention, but not quite understand how do I use it? Can you please provide any simple usage example?

Thank you!

**Comment by ilian6806** [comment, marked as answer]
Hello @susanin-ivan,

Thanks for the feedback! I will provide in my readme some basic usage and examples! 

Basically when SD creates faces and eyes sometimes it makes mistakes. So I created this extension to mark the eyes or the whole face and redraw it with a new prompt. Also if you want random eyes for example, you can create *my_eyes.txt* for example file in /wildcards folder (in extension folder) and use '__my_eyes__ eyes' for your prompt. It will take a random name from the file in replace the placeholder. If you want to iterate over each placeholder in the text file name must end with *_each*.

I created a little video for this basic usages -> https://www.youtube.com/watch?v=Q5PIFd7XsjM

I hope this helps!

**Reply by susanin-ivan** [reply]
Thank you a lot for this example!!!

No more struggling with eyes/face selection for inpainting :)))
